android/app/src/main/java/io/mosip/residentapp/InjiVciClientModule.java

```diff
@@ -42,19 +42,33 @@ public String getName() {
     public void requestCredential(ReadableMap issuerMetaData, String jwtProofValue, String accessToken, Promise promise) {
         try {
             CredentialFormat credentialFormat;
+            IssuerMetaData constructedIssuerMetadata ;
             switch (issuerMetaData.getString("credentialFormat")) {
                 case "ldp_vc":
                     credentialFormat = CredentialFormat.LDP_VC;
-                    break;
-                default:
-                    credentialFormat = CredentialFormat.LDP_VC;
-            }
-            CredentialResponse response = vciClient.requestCredential(new IssuerMetaData(
+                    constructedIssuerMetadata =  new IssuerMetaData(
                             issuerMetaData.getString("credentialAudience"),
                             issuerMetaData.getString("credentialEndpoint"),
                             issuerMetaData.getInt("downloadTimeoutInMilliSeconds"),
                             convertReadableArrayToStringArray(issuerMetaData.getArray("credentialType")),
-                            credentialFormat), new JWTProof(jwtProofValue)
+                            credentialFormat,null,null);
+                    break;
+                case "mso_mdoc":
```

Comment:
Lets move these to constants

Reply:
Modified to iff else for removing harcoding as switch case is having issues using Credential format enum constant values